odoo/tools/translate.py

```diff
@@ -842,18 +842,19 @@ def add_entry(self, modules, tnrs, source, trad, comments=None):
         entry.comment = "module%s: %s" % (plural, ', '.join(modules))
         if comments:
             entry.comment += "\n" + "\n".join(comments)
-
-        code = False
-        for typy, name, res_id in tnrs:
-            if typy == 'code':
-                code = True
-                res_id = 0
-            if isinstance(res_id, int) or res_id.isdigit():
-                # second term of occurrence must be a digit
-                # occurrence line at 0 are discarded when rendered to string
-                entry.occurrences.append((u"%s:%s" % (typy, name), str(res_id)))
+        occurrences = set()
+        for type_, *ref in tnrs:
```

Comment:
I would unpack everything here, it would avoid unpacking `ref` later and the awkward `join`